src/interfaces.ts

```diff
@@ -40,6 +42,8 @@ export interface Options {
   mirostat_eta: number
   penalize_newline: boolean
   stop: string[]
+  rope_frequency_base: number
+  rope_frequency_scale: number
```

Comment:
```suggestion
```

These were also omitted because they will be deprecated and handled by Ollama soon.